src/runtime/job.cpp

```diff
@@ -824,8 +830,18 @@ static void launch(JobTable *jobtable) {
     } else {
       err = std::make_unique<NullBuf>();
     }
+
+    auto stdout_teefiles = std::vector<std::unique_ptr<std::ofstream>>();
+    for (auto tee_path : task.job->tee_out) {
+      stdout_teefiles.push_back(std::make_unique<std::ofstream>(tee_path, std::ios::out | std::ios::app));
```

Comment:
hmm, we should talk about if these should be append only or not. I think I'm inclined to say they should not be appended to.

Reply:
Over-the-wall discussion about storing these all to the JobTable has resolved this.